### DISCUSSION TITLE
5.x如何在展示图例的时候设置类似4.x中offset配置项的效果？

### DISCUSSION THREAD

**Discussion by voiddiddvue** [antvis/G2#5637] [Q&A]
5.x版本去掉了legend.offsetX和offsetY功能，这样对定义图例位置很不友好，有方法实现吗

**Comment by pearmini** [comment]
使用 transform 属性。

```js
chart.interval().legend('color', { transform:`translate(100, 0)` })
```

**Reply by voiddiddvue** [reply]
好的，感谢，另外请问为什么在使用圆形分面时，设置title无效，
即基于这个例子https://g2.antv.antgroup.com/examples/composition/facet#circle

`const facetCircle = chart.facetCircle()
    .axis('x', {
      labelFill: 'red'
    })
    .title('123')  //标题没有效果
    .data(data).encode('position', 'month');`

**Reply by pearmini** [reply]
这个 title 是分面整体的 title，还是每个 view 的titlte

**Reply by voiddiddvue** [reply]
整体的title

**Reply by guojingwen** [reply]
不生效

**Reply by tiger5wang** [reply]
请问你的使用 transform 属性生效了吗，我这里不生效呢
